dlib/common/include/text.txt

```diff
@@ -0,0 +1 @@
+
```

Comment:
This file is important because git will delete the "common/include" folder otherwise and they have to exist before build.